docker/standalone/start.sh

```diff
@@ -24,7 +24,7 @@
 pushd /
 ./docker-entrypoint.sh cassandra
 popd
-sleep 20
+sleep 40
```

Comment:
How about someway to health check cassandra (e.g. via `cqlsh -e 'desc system'`) instead of fixed sleep?